bot/cogs/moderation/infractions.py

```diff
@@ -243,6 +247,20 @@ def __init__(self, bot: Bot):
         action = ctx.guild.ban(user, reason=reason, delete_message_days=0)
         await self.apply_infraction(ctx, infraction, user, action)
 
+        if infraction.get('expires_at') is not None:
+            log.trace(f"Ban isn't permanent; user {user} won't be unwatched by Big Brother.")
+            return
+
+        bb_cog = self.bot.get_cog("Big Brother")
+        if not bb_cog:
+            log.trace(f"Big Brother cog not loaded; perma-banned user {user} won't be unwatched.")
```

Comment:
While I understand that this is part of the trace logging flow, if this situation is encountered then this should be issued at the error level, not a trace.

Reply:
Why?

Reply:
Corrected in the most recent commit

Reply:
Why wouldn't we want to be notified that the big brother cog isn't loaded?

Reply:
For the purpose of the trace logic it makes no difference if this is an error or a trace, but for the sake of knowing that something isn't quite right with one of our cogs, it makes total sense for it to be an error.  I just didn't think about it when writing it.

Reply:
If that's the case then IMO a warning is more appropriate. If there's ever a legitimate reason to unload it, even temporarily, then it'd be strange to be greeted with an error when nothing is actually wrong with the code. If you don't want it to be unloaded, consider adding it to the blacklist in the extensions cog.

Reply:
Fair point, a warning makes sense to me.  @sco1 that work for you?